nvi-evaluator/src/main/java/no/sikt/nva/nvi/evaluator/EvaluateNviCandidateHandler.java

```diff
@@ -58,30 +60,47 @@ protected Void processInputPayload(EventReference input,
             var candidateType = calculator.calculateNvi(jsonNode);
             handleCandidateType(input.getUri(), candidateType);
         } catch (Exception e) {
-            var msg = "Failure while calculating NVI Candidate: %s, ex: %s, msg: %s"
-                          .formatted(input.getUri(), e.getClass(), e.getMessage());
+            var msg = "Failure while calculating NVI Candidate: %s, ex: %s, msg: %s".formatted(input.getUri(),
+                                                                                               e.getClass(),
+                                                                                               e.getMessage());
             LOGGER.error(msg, e);
             queueClient.sendDlq(msg);
         }
         return null;
     }
 
+    private static CandidateDetails extractCandidateDetails(NonNviCandidate candidateType) {
```

Comment:
I would call this method something with "create" instead because you are creating a new CandidateDetails object. For example createCandidateDetails?